batch/batch/worker_config.py

```diff
@@ -98,6 +98,11 @@ def __init__(self, config):
         self.cores = instance['cores']
         self.preemptible = instance['preemptible']
 
+        data_disk = [d for d in self.disks if not d['boot']][0]
+        assert data_disk
+        self.local_ssd_data_disk = (data_disk['type'] == 'local-ssd')
+        self.data_disk_size_gb = data_disk['size']
```

Comment:
Can we change this to asserting there are two disks and the first one is the boot disk, and just take the second one?  This code needs to change if there are >2 disks.